core/src/main/java/org/apache/calcite/rel/logical/LogicalJoin.java

```diff
@@ -170,36 +170,6 @@ public static LogicalJoin create(RelNode left, RelNode right, List<RelHint> hint
         variablesSet, joinType, semiJoinDone, systemFieldList);
   }
 
-  @Deprecated // to be removed before 1.23
-  public static LogicalJoin create(RelNode left, RelNode right,
-      RexNode condition, Set<CorrelationId> variablesSet, JoinRelType joinType) {
-    return create(left, right, ImmutableList.of(), condition, variablesSet,
-        joinType, false, ImmutableList.of());
-  }
-
-  @Deprecated // to be removed before 1.23
-  public static LogicalJoin create(RelNode left, RelNode right,
-      RexNode condition, Set<CorrelationId> variablesSet, JoinRelType joinType,
-      boolean semiJoinDone, ImmutableList<RelDataTypeField> systemFieldList) {
-    return create(left, right, ImmutableList.of(), condition, variablesSet,
-        joinType, semiJoinDone, systemFieldList);
-  }
-
-  @Deprecated // to be removed before 2.0
-  public static LogicalJoin create(RelNode left, RelNode right,
-      RexNode condition, JoinRelType joinType, Set<String> variablesStopped,
-      boolean semiJoinDone, ImmutableList<RelDataTypeField> systemFieldList) {
-    return create(left, right, condition, CorrelationId.setOf(variablesStopped),
-        joinType, semiJoinDone, systemFieldList);
-  }
-
-  @Deprecated // to be removed before 2.0
-  public static LogicalJoin create(RelNode left, RelNode right,
-      RexNode condition, JoinRelType joinType, Set<String> variablesStopped) {
```

Comment:
These 2 create method are also removed, because i think the `create#` method is low frequency used API (usuallu we use the `RelBuilder`).